tests/v2/validation/provisioning/rke1/config.go

```diff
@@ -0,0 +1,25 @@
+package rke1
```

Comment:
I think it would be better to just have one config.go file, and have it be in the v2/validation/provisioning directory above rke1 and rke2. That way we don't have two of the same file in each in package. And baseClusterName for each pack (rke1 and rke2) can have the correct prefix. Makes sense?

Reply:
No problem, I can remove it. I had added it in this PR, but I'll revert those changes 👍🏿 .